edit/service.vue

```diff
@@ -75,6 +107,12 @@ export default {
       total:   0,
     };
 
+    const harvesterAddOnConfig = {};
```

Comment:
Can the harvester specific code here be gated on `showHarvesterAddOnConfig` (might need to move this into the fetch)?

Reply:
Done. I add a component named "HarvesterServiceAddOnConfig". When showHarvesterAddOnConfig is true, shows the Add-on config.